reliability/README.md

```diff
@@ -1,81 +1,205 @@
-OpenShift V4 Reliability 
-===========================
+# OpenShift V4 Reliability
+## Introduction
 Software reliability testing is a field of software testing that relates to testing a software's ability to function, given environmental conditions, for a particular amount of time.
 Openshift Reliability testing is an operational testing scheme that uses a baseline work efficiency specification to evaluate the stability of openshift system in the given amount of time. The purpose is to discover problems in functionality. The baseline work efficiency specification was made up of daily tasks such as applications developing, hosting and scaling. 
-<br/>
 
-Configure (WIP)
-===========================
-Tasks are classified as minutely, hourly, weekly or monthly.  To each of these intervals, a real time interval is assigned.  This is done to allow assigning activities with a natural spacing while allowing for faster simulation of the activities.  Below is a sample working configuration. Note that not all fields are currently supported as this tool is a work in progress.  In particular the percentages for performing activities on a subset of the resources is not yet implemented.
+## Configuration
+Example [config](https://github.com/openshift/svt/blob/master/reliability/config/example_reliability.yaml). Below sections explains each part of the configuration file.
+
+### Files needed
+
+If you installed your cluster with [Flexy-install](https://mastern-jenkins-csb-openshift-qe.apps.ocp4.prod.psi.redhat.com/job/ocp-common/job/Flexy-install/), download kubeconfig, users.spec and kubeadmin-password files. If you installed cluster in other way, prepare the above files accordingly.
+
+Replace kubeconfig kubeadmin_file and user_file in your config file.
```

Comment:
Think it should be 
`kubeconfig, kubeadmin_password and user_file`

Reply:
I mean these 3 files in the screenshot below. 
<img width="503" alt="Screen Shot 2021-07-13 at 11 06 28 AM" src="https://user-images.githubusercontent.com/81630527/125384244-34490900-e3cb-11eb-9b06-71c7b00bb786.png">

Why do you say kubeconfig, kubeadmin_password and user_file? Where do you see them or do you just mean to use amore common names?

Reply:
Just to have a more common names. What you have now "kubeadmin_file" is a little confusing and I wasn't 100% sure what you meant. I think changing "kubeadmin_file" to kubeadmin-password would make it a little more clear.

Reply:
make sense, I'll update the config file.